package.json

```diff
@@ -29,7 +29,14 @@
     "dist/*.js"
   ],
   "scripts": {
+    "autobuild": "rollup -c -w",
+    "build": "rollup -c",
+    "dev": "concurrently --kill-others \"npm run autobuild\" \"cross-env NODE_ENV=test karma start ---auto-watch --no-single-run\"",
     "docs": "cd docs && npm install && npm run build && mkdir -p ../dist && cp -r build ../dist/docs",
+    "lint-js": "eslint samples/**/*.html samples/**/*.js src/**/*.js test/**/*.js",
+    "lint-tsc": "npx tsc",
+    "lint": "concurrently \"npm run lint-js\" \"npm run lint-tsc\"",
```

Comment:
according to the docs: https://www.npmjs.com/package/concurrently

this could be simplified to `concurrently \"npm:lint-*\"`

Reply:
done